quantumdb-core/src/main/java/io/quantumdb/core/versioning/QuantumTables.java

```diff
@@ -20,20 +20,20 @@
 			"CREATE TABLE quantumdb.config (name VARCHAR(255) NOT NULL, value VARCHAR(255) NOT NULL, PRIMARY KEY (name));",
 
 			// Creates the "changelog" table which will store all the individual (schema) operations, and their ordering.
-			"CREATE TABLE quantumdb.changelog (version_id VARCHAR(10) NOT NULL, parent_version_id VARCHAR(10), operation_type VARCHAR(16), operation TEXT, PRIMARY KEY (version_id));",
+			"CREATE TABLE quantumdb.changelog (version_id VARCHAR(255) NOT NULL, parent_version_id VARCHAR(255), operation_type VARCHAR(255), operation TEXT, PRIMARY KEY (version_id));",
```

Comment:
These increased numbers are used in the "cleanup" pull request where I changed it so that the last VersionId is the same as the ChangeSetId

Reply:
I would advice against this. 

- First of all this breaks databases that were already initialized with quantumdb (their tables won't see these changes).
- This seems to be a change you don't strictly need for the cleanup PR.
- Don't change the meaning of `versionId`, but instead use the `quantumdb.changelog` table where you're parsing user input where you're expecting version ids. This table already has a mapping for every `changeSetId` <--> `versionId`.